swagger-coding-style.md

```diff
@@ -272,14 +272,12 @@ by any API that does not explicitly override this value.
 
 ### Use well-defined parameter types
 
-Parameters should have well-defined type and format information.
-Only use combinations of `type` and `format` defined in the
-[Swagger Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types)
-Parameter types are further constrained by their "in" property, as specified in
-[Swagger Specification, Parameter Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameter-object).
+Parameters should have well-defined type and format information. Only use combinations of `type` and `format` defined in the [Swagger Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types) Parameter types are further constrained by their "in" property, as specified in [Swagger Specification, Parameter Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameter-object).
```

Comment:
Please update the Swagger text/links to OpenAPI 3.0.

Reply:
Updated to current spec version 3.0.2